CHANGES.md

```diff
@@ -1,3 +1,30 @@
+## edge-19.6.2
+
+* CLI
+  * Added the `--linkerd-cni-enabled` flag to the `install` subcommands so that
+    `NET_ADMIN` capability is omitted from the CNI-enabled control plane's PSP
+* Controller
+  * Fixed security context values so that injection does not fail on
```

Comment:
Can we change this to something like _Defined least privilege default security context values for the proxy container so that auto-injection does not fail on..._?

Reply:
Wording a bit differently but sure!